runtime/src/bank.rs

```diff
@@ -6688,6 +6697,10 @@ impl Bank {
         );
     }
 
+    fn adjust_sysvar_rent_epoch(&self, account: &mut AccountSharedData) {
+        account.set_rent_epoch(solana_sdk::rent_collector::RENT_EXEMPT_RENT_EPOCH);
+    }
+
```

Comment:
This function seems to be called from a single call site.  Why the new function is even needed?.. Couldn't the single statement of the function body be added at the call site of the function instead?

Reply:
I am trying to mirror what we have done for "adjust_sysvar_balance_for_rent()", more like a "documentation" than logic need. 

I am fine with inlining it too.